app/src/plugins/sd-ct-oauth-plugin/auth.router.js

```diff
@@ -865,6 +931,10 @@ module.exports = (plugin, connection, generalConfig) => {
     ApiRouter.get('/twitter', setCallbackUrl, API.twitter);
     ApiRouter.get('/twitter/callback', API.twitterCallback, API.updateApplications);
 
+    ApiRouter.get('/apple', setCallbackUrl, API.apple);
+    ApiRouter.post('/apple/callback', API.appleCallback, API.updateApplications);
+    ApiRouter.get('/apple/token', API.appleToken, API.generateJWT);
```

Comment:
is `API.generateJWT` needed? looks like there's a token being generated already at the end of the `API.appleToken` method

Reply:
good catch - I actually want to keep the call to generateJWT, and make the custom code match that.